templates/getting_started_typescript/package.json

```
@@ -0,0 +1,22 @@
+{
+    "name": "getting-started-typescript",
+    "version": "0.0.1",
+    "description": "This is an example of an Apify actor.",
+    "dependencies": {
+        "apify": "^2.3.0"
+    },
+    "devDependencies": {
+        "@apify/tsconfig": "^0.1.0",
+        "ts-node": "^10.3.0",
+        "typescript": "^4.4.4"
```

let's bump to latest
```suggestion
        "typescript": "^4.6.3"
```